src/Domain/Entities/Order.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Domain.Common;
+
+namespace Domain.Entities
+{
+    class Order : BaseEntity
+    {
+        public int UserId { get; set; }
+        public string ShippingAddress { get; set; }
+        public string Phone { get; set; }
+        public decimal DeliveryCharge { get; set; }
+        public int Status { get; set; }
+        public DateTime AddedOn { get; set; }
```

Comment:
Should this be needed as we have CreatedAt in BaseEntity @deba1 @kanismim ?

Reply:
No. It's not needed anymore.